src/app/common/CopyButton.vue

```diff
@@ -94,23 +94,22 @@ async function copy(event: Event, copyToClipboard: (text: string) => Promise<boo
 }
 
 .copy-button {
-  --tooltip-text-color: var(--white);
-  --tooltip-background-color: var(--grey-600);
+  --tooltip-background-color: $kui-color-background-neutral-stronger;
```

Comment:
issue: when setting a CSS custom property to a SCSS variable, you **must** use interpolation so that the value is compiled. Comment applies to whole app/PR
```suggestion
  --tooltip-background-color: #{$kui-color-background-neutral-stronger};
```

Reply:
Fixed everywhere. It’s the consequence of me replacing most of these with a regexp-based search-and-replace.